common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -104,6 +105,91 @@ def test_courseware(self):
 
         # Visit courseware as a student.
         self.courseware_page.visit()
-
         # Problem name should be "TEST PROBLEM 2".
         self.assertEqual(self.problem_page.problem_name, 'TEST PROBLEM 2')
+
+
+class CoursewareMultipleVerticalsTest(UniqueCourseTest):
+    """
+    Test courseware with multiple verticals
+    """
+    USERNAME = "STUDENT_TESTER"
+    EMAIL = "student101@example.com"
+
+    def setUp(self):
+        super(CoursewareMultipleVerticalsTest, self).setUp()
+
+        self.courseware_page = CoursewarePage(self.browser, self.course_id)
+
+        self.course_outline = CourseOutlinePage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        # Install a course with sections/problems, tabs, updates, and handouts
+        course_fix = CourseFixture(
+            self.course_info['org'], self.course_info['number'],
+            self.course_info['run'], self.course_info['display_name']
+        )
+
+        course_fix.add_children(
+            XBlockFixtureDesc('chapter', 'Test Section 1').add_children(
+                XBlockFixtureDesc('sequential', 'Test Subsection 1').add_children(
+                    XBlockFixtureDesc('problem', 'Test Problem 1', data='<problem>problem 1 dummy body</problem>'),
+                    XBlockFixtureDesc('html', 'html 1', data="<html>html 1 dummy body</html>"),
+                    XBlockFixtureDesc('problem', 'Test Problem 2', data="<problem>problem 2 dummy body</problem>"),
+                    XBlockFixtureDesc('html', 'html 2', data="<html>html 2 dummy body</html>"),
+                ),
+                XBlockFixtureDesc('sequential', 'Test Subsection 2'),
+            ),
+        ).install()
+
+        # Auto-auth register for the course.
+        AutoAuthPage(self.browser, username=self.USERNAME, email=self.EMAIL,
+                     course_id=self.course_id, staff=False).visit()
+        self.courseware_page.visit()
+        self.course_nav = CourseNavPage(self.browser)
+
+    def test_tab_position(self):
+        # test that using the position in the url direct to correct tab in courseware
+        self.course_nav.go_to_section('Test Section 1', 'Test Subsection 1')
+        subsection_url = self.courseware_page.get_active_subsection_url()
+        url_part_list = subsection_url.split('/')
+        self.assertEqual(len(url_part_list), 9)
```

Comment:
Can you leave an empty line after each of the asserts? It looks a bit cluttered right now.